Add files via upload

rcread opened the pull request:
Added header component

rcread committed:
Add files via upload

magda-dudzic commented on index.html:
```diff
@@ -0,0 +1,106 @@
+            </div>
+        </div>
+        
+        <div class="menu-bar">
```
`nav` element would be better - more info here https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav 
It doesn't make a visual difference, it's more about the semantics.

rcread committed:
Fonts and Images upload

magda-dudzic commented on main.css:
```diff
@@ -0,0 +1,227 @@
+    margin: 0px;
+}
+
+header {
```
Please don't style by element names (unless it's a global style, like the one for body). In bigger projects than yours it might cause issues when styling. Try to always style by classname.

rcread replied:
Have kept the below because it seems to have an auto margin I want to get rid of - or should I add a class name to the body? 

body {
     margin: 0px;
}

magda-dudzic commented on main.css:
```diff
@@ -0,0 +1,227 @@
+    padding: 4px 16px;
+    margin-right: 32px;
+}
+/* Is this ok to make the breakpoint 1023 instead of 1024? Same for a few media queries below. No opposite for display:none to make it min-width:1024*/
```
well, in theory you can create any breakpoint you want but my question is - why 1023? Width <= 1024 is usually a tablet width, so you'd wanna style either <= 1024 for tablet, or >1024 for desktop

rcread replied:
I used 1023 because I wanted it to be below 1024px and not including 1024. I have only used min-width and max-width before so i'll try width <= now.

rcread replied:
Have amended it now to use display block on the media query and display none without media query.

magda-dudzic commented on main.css:
```diff
@@ -0,0 +1,227 @@
+
+.menu-bar-l {
+
+}
```
try not to leave empty fields like that

rcread replied:
Removed!

magda-dudzic commented on main.css:
```diff
@@ -0,0 +1,227 @@
+    }
+}
+
+@media (min-width:1264px) {
```
why those two breakpoints with the same style?

rcread replied:
I did that so it was only visible between 1024px and 1264px

magda-dudzic commented on main.css:
```diff
@@ -0,0 +1,227 @@
+}
+
+.menu-bar-xl {
+    margin:32px 108px 12px;
```
there's no space between margin and 32 - you can add a prettier plugin to VS code which will format your files on save for instance

magda-dudzic commented on index.html:
```diff
@@ -0,0 +1,106 @@
+                    <img class="search-icon" src="img/search.svg"/>
+                </a>
+            </div>
+            <img class="nav-column tws-logo" src="img/tws-logo.png"/>
```
Logo in the menu should always be a link to the homepage ('/')

magda-dudzic replied:
I can also see the logo is poor quality - when exporting from Figma you can choose option 'x2' for size

magda-dudzic reviewed (commented):
You can see the difference in how the menu looks on production vs how it looks in your project. 
<img width="1187" alt="Zrzut ekranu 2022-11-8 o 11 31 59" src="https://user-images.githubusercontent.com/87694556/200542186-5d4870bb-9920-4325-adfe-7501c06761ff.png">
<img width="1189" alt="Zrzut ekranu 2022-11-8 o 11 32 06" src="https://user-images.githubusercontent.com/87694556/200542212-4c61f608-b52e-45b5-a58d-bb9d542fb1bb.png">

magda-dudzic commented:
> You can see the difference in how the menu looks on production vs how it looks in your project. <img alt="Zrzut ekranu 2022-11-8 o 11 31 59" width="1187" src="https://user-images.githubusercontent.com/87694556/200542186-5d4870bb-9920-4325-adfe-7501c06761ff.png"> <img alt="Zrzut ekranu 2022-11-8 o 11 32 06" width="1189" src="https://user-images.githubusercontent.com/87694556/200542212-4c61f608-b52e-45b5-a58d-bb9d542fb1bb.png">

I think you forgot about letter spacing, the colour's not right either

rcread committed:
Adding scss file and amendments 

Included the main.css file but presumably this just needs to be removed as I've started with a new scss file 'style' instead. Link from index.html to main.css has been commented out.

rcread committed:
update to TWS logo height from 6px to 6.5px

rcread committed:
Updates to scss file, include-media

Problems with include-media working in CSS file, appears to work OK in SCSS file.

rcread committed:
upload of package.json

rcread committed:
Delete style.css

rcread committed:
Delete style.css.map

rcread committed:
Delete style.scss

rcread committed:
SCSS updates - added in include-media

Issues with CSS file error identifier expected

rcread committed:
Updates to SCSS file

Deleted and recreated CSS file, removed !default and resolved errors

rcread committed:
Updated header to include container

rcread committed:
Add files via upload

rcread committed:
CSS changes

Updating spacing between menu-items - still need to figure out auto spacing for responsive design

rcread committed:
fixed styles in the menu
